dnf/cli/commands/shell.py

```diff
@@ -113,8 +113,8 @@ def onecmd(self, line):
                     cmd.cli.demands = copy.deepcopy(self.cli.demands)
                     cmd.configure()
                     cmd.run()
-                except:
-                    pass
+                except Exception as e:
+                    logger.error(' '.join([_("Error:"), e.value]))
```

Comment:
1. "Error:" message is translated to other languages. "Error: %s" not.
2. And I use join because someone told me: Python programmer prefers ' '.join() instead operator +.